app/players/player.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
```

Comment:
The `get_rating` method is annotated to return `None`, but in subclasses, it returns an `int`. You should change the return type annotation to `int` to match the expected return type.